packages/cli/src/networks/ropsten.ts

```diff
@@ -0,0 +1,13 @@
+import {ropstenChainConfig} from "@chainsafe/lodestar-config/networks";
+
+export const chainConfig = ropstenChainConfig;
+
+/* eslint-disable max-len */
+
+export const depositContractDeployBlock = 12269949;
+export const genesisFileUrl =
+  "https://raw.githubusercontent.com/eth-clients/merge-testnets/main/ropsten-beacon-chain/genesis.ssz";
+export const bootnodesFileUrl =
+  "https://raw.githubusercontent.com/eth-clients/merge-testnets/main/ropsten-beacon-chain/bootstrap_nodes.txt";
+
+export const bootEnrs = [];
```

Comment:
found this value in https://github.com/eth-clients/merge-testnets/tree/main/ropsten-beacon-chain

```
# EF bootnode
  - "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk"
# Teku bootnode
  - "enr:-KG4QMJSJ7DHk6v2p-W8zQ3Xv7FfssZ_1E3p2eY6kN13staMObUonAurqyWhODoeY6edXtV8e9eL9RnhgZ9va2SMDRQMhGV0aDKQS-iVMYAAAHD0AQAAAAAAAIJpZIJ2NIJpcIQDhAAhiXNlY3AyNTZrMaEDXBVUZhhmdy1MYor1eGdRJ4vHYghFKDgjyHgt6sJ-IlCDdGNwgiMog3VkcIIjKA"
```

Reply:
bootnodesFileUrl should pick it up